appside/frames/body.js

```diff
@@ -215,15 +218,39 @@ class BodyMapFrame extends Frame {
             $(label).text(stmt);
             right.appendChild(label);
             right.appendChild(document.createElement('br'));
+
+            this.user_input.set(name, 'false'); // all unchecked
         }
 
+        // next will be implemented in navigator?
+        let next = document.createElement('button');
+        $(next).attr('class', 'bodymap_button');
+        $(next).text('Next');
+        $(next).click(function() {
+            var choices = document.getElementsByTagName('input');
+            for (let each of choices) {
+                if (each.checked) {
```

Comment:
Also important to update the unchecked boxes. Use case illustrating why:

1. User navigates to page 1
2. User checks all the boxes on page 1
3. User navigates away from page 1 and model is updated to reflect that all boxes on page 1 are checked.
4. User does some other things
5. User navigates BACK to page 1 and unchecks some of the boxes.
(Need to explicitly provide an update saying that those boxes are now unchecked.)

Reply:
This can be fixed by making sure checkbox is unchecked/checked and setting value accordingly.